app/jobs/failing_job.rb

```diff
@@ -0,0 +1,9 @@
+class FailingJob < ApplicationJob
+  queue_as :default
+
+  def perform(test_data)
+    with_raven_context(test_data: test_data) do
+      raise "forced test exception with context"
+    end
+  end
+end
```

Comment:
I am not sure if we still need to include this but it was helpful to test on staging. Let me know what you think?